packages/dashboard/src/components/sidePanel/sections/chartSettings.tsx

```diff
@@ -1,11 +1,124 @@
 import React from 'react';
-import { ExpandableSection } from '@cloudscape-design/components';
+import { Checkbox, ExpandableSection, Input, Select } from '@cloudscape-design/components';
 import ExpandableSectionHeader from '../shared/expandableSectionHeader';
+import { capitalizeFirstLetter, useInput } from '../utils';
+import './styles.css';
+import SettingTile from '../shared/settingTile';
+import { LEGEND_POSITION } from '@synchro-charts/core';
+
+const ChartTitleSetting = () => {
+  const [title, updateTitle] = useInput<string>('title');
```

Comment:
Is <string> necessary? Does the hook not infer the string type from 'title'?

Reply:
No. The hook only reads a string attribute from an object. It does not know return type of it.